spec/features/ncr_proposal_spec.rb

```diff
@@ -87,5 +87,49 @@
       expect(current_path).to eq("/carts/#{Cart.last.id}")
       expect(page).to have_content("RWA Number")
     end
+
+    it "can be restarted if pending" do
+      proposal = FactoryGirl.build(:proposal_form)
+      proposal.requester = requester
+      cart = proposal.create_cart
```

Comment:
I wanted the properties from the factorygirl proposal form. If I just used `:cart`, I wouldn't get those, I don't think.